actors.md

```diff
@@ -474,7 +475,7 @@ func CommitSector(comm Commitment, proof *SealProof) SectorID {
 	miner.Collateral -= coll
 	miner.ActiveCollateral += coll
 
-	sectorId = miner.Sectors.Add(commR)
+	miner.Sectors.Add(commR)
```

Comment:
we should probably pass the sectorID as a parameter here, so it can insert it into the right place. Also, do we just want to store commR? I think we want to store commD as well for payments.

Reply:
> we should probably pass the sectorID as a parameter here, so it can insert it into the right place. Also, do we just want to store commR?

Good call. We do want the CommD (for payments) and we do want the SectorID (so we insert things at the correct indices).

How shall I represent this in psuedocode? How about:

```
miner.Sectors.Add((sectorID, commR, commD))
```

Reply:
Just pushed update to pseudo code + comment.